SPECS/python-tensorboard/python-tensorboard.spec

```diff
@@ -0,0 +1,110 @@
+%global pypi_name tensorboard
+%global _description %{expand:
+TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs}
+%define _enable_debug_package 0
+%global debug_package %{nil}
+
+Summary:        TensorBoard is a suite of web applications for inspecting and understanding your TensorFlow runs and graphs
+Name:           python-%{pypi_name}
+Version:        2.11.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/tensorflow/tensorboard
+Source0:        https://github.com/tensorflow/tensorboard/archive/refs/tags/%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+Source1:        %{name}-%{version}-cache.tar.gz
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-pip
+BuildRequires:  python3-wheel
+BuildRequires:  python3-six
+BuildRequires:  bazel = 5.3.0
+BuildRequires:  python3-tf-nightly = 2.11.0
+BuildRequires:  gcc
+BuildRequires:  build-essential
+BuildRequires:  protobuf
+BuildRequires:  zlib
+BuildRequires:  python3-virtualenv
+ExclusiveArch:  x86_64
+
+
+%description %{_description}
+
+%package -n python3-%{pypi_name}
+Summary:        %{summary}
+Requires:   python3-absl-py
+Requires:   python3-google-auth-oauthlib = 0.5.3
+Requires:   python3-google-auth = 2.6.6
+Requires:   python3-grpcio 
+Requires:   python3-markdown
+Requires:   python3-numpy
+Requires:   python3-protobuf = 3.17.3
+Requires:   python3-requests = 2.27.1
+Requires:   python3-setuptools
+Requires:   python3-tensorflow-estimator = 2.11.0
```

Comment:
Why do we need to be so strict on these package versions? This may cause upgrade issues (a different package may require a newer versions of one of these for instance) and leave us vulnerable to security bugs fixed in newer versions of these packages.

Reply:
![image](https://user-images.githubusercontent.com/106988478/209240320-ad7232e9-0aa5-4d07-90f7-739e05fc8034.png)
 yes this is locked.